src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -138,13 +138,20 @@ impl CloudHypervisorInner {
         // Start by adding the default set of kernel parameters.
         let mut params = KernelParams::new(enable_debug);
 
+         #[cfg(target_arch = "x86_64")]
+        let console_param_debug = KernelParams::from_string("console=ttyS0,115200n8");
+
+        #[cfg(target_arch = "aarch64")]
+        let console_param_debug = KernelParams::from_string("console=ttyAMA0,115200n8");
+
+
         let mut rootfs_param = KernelParams::new_rootfs_kernel_params(rootfs_driver, rootfs_type)?;
 
         let mut console_params = if enable_debug {
             if confidential_guest {
```

Comment:
Not related to this PR, but would you be able to look at raising a PR to allow `confidential_guest=true` on aarch64 with runtime-rs and CH?

Reply:
There is no confidential computing support in any of VMM on aarch64 for now. So, I think it's not the time to enable confidential guest.